docs/tutorials/ServerModule/ServerModule.md

```diff
@@ -3,77 +3,94 @@ uid: GettingStarted.ServerModule
+  - [The ModuleConfig.cs - File](#the-moduleconfigcs---file)
+  - [The ModuleConsole.cs - File](#the-moduleconsolecs---file)
 
-    * [ModuleController.cs](#moduleControllerFile)
-    * [ModuleConfig.cs](#moduleConfigFile)
-    * [ModuleConsole.cs](#moduleConsoleFile)
-
-## The ModuleController.cs - File <a id="moduleControllerFile" />
-
-The ModuleController.cs-File is the key point of your module. Here all the components of your module came together and you are responsible to initialize, start and stop them in the right way. Because every ServerModule can have its own architecture there is not _the correct way_ to do so, but there are some _usually points_ a ModuleController.cs-File covers. These points are:
+## The ModuleController.cs 
+The ModuleController.cs-File is the key point of your module. Here all the components of your module come together and you are responsible to initialize, start and stop them in the right way. Because every ServerModule can have its own architecture, there is not _the correct way_ to do so, but there are some _usual points_ a ModuleController.cs-File covers. These points are:
 
 1. Import the global components your ServerModule needs
 2. Register imported _global_ components to the internal container
 3. Resolve the desired components from the container and **start** them
 4. Stop the started components when the ServerModule is stopped
-5. (Export and Import facades -> this topic is covered in [this guide](xref:FacadeGuide))
+5. Export and Import facades -> this topic is covered in [this guide](Facades.md)
+
+Now we will look at examples for these points. But first create your your class implementing `ServerModuleBase`. If your ServerModule exports facades, use `ServerModuleFacadeControllerBase` instead and specifiy those facades using `IFacadeContainer<TFacade>`.
 
-Now we will look at examples for these points. But first create your your class with the ServerModuleAttribute.cs and the ServerModuleBase.cs.
 For the following properties and attributes reference these files:
 
 * Moryx.dll
 * Moryx.Runtime.dll
 
 ````cs
-[ServerModule(ModuleName)]
 [Description("Example description")]
 public class ModuleController : ServerModuleBase<ModuleConfig>
 {
     internal const string ModuleName = "ExampleName";
+    
     /// <summary>
     /// Name of this module
     /// </summary>
     public override string Name
     {
         get { return ModuleName; }
     }
-.
-.
-.   
+    
+    ...
 ````
 
-As example for the first point we import the ResourceManagement and the ProductManagement. We do so by simply write them as public properties, the global DI container will do the rest. (The RequiredModuleApi-Attribute is described [here](xref:FacadeGuide))
+As example for the first point we import the ResourceManagement and the ProductManagement. We do so by simply write them as public properties, the global DI container will do the rest. (The RequiredModuleApi-Attribute is described [here](Facades.md))
```

Comment:
- As an example for the first bullet point, ...
- ... simply adding them as ...
-